apis/python/tests/test_shape.py

```diff
@@ -679,10 +674,7 @@ def _check_ndarray(ndarray, has_shapes, expected_shape):
     dict_output["ms"]["raw"]["var"]["uri"] = "test"
     dict_output["ms"]["raw"]["X"]["data"]["uri"] = "test"
 
-    if has_shapes:
-        var_max_domain_hi = 9223372036854773968
-    else:
-        var_max_domain_hi = 9223372036854773758
+    var_max_domain_hi = 9223372036854773968 if has_shapes else 9223372036854773758
```

Comment:
Could we assign this magic numbers in a `global` constant?

Reply:
I agree that the tests need cleanup - for example, they don't use type hints. I have had to disable quite a few of the linter rules for the tests directory.

My preference is to just focus on the `src` (i.e. shipped code) in these PRs, and later work on incrementally fixing tests.  Please LMK your thoughts.

The above code is simply an auto-fix - the constants were from the original shaping work done in 1.15